doc/api/api_changes/2017-12-20-TH.rst

```diff
@@ -0,0 +1,5 @@
+Deprecations
+````````````
```

Comment:
I think underscores, right?

Reply:
All the other files for deprecations in the folder use this format.